reference/6/About/about_Special_Characters.md

```diff
@@ -66,6 +68,25 @@ The output from this command is as follows:
 
 back out
 
+ESCAPE (`e)
+The escape character is most commonly used to specify virtual terminal
+sequences to modify the color of text in addition to other text attributes
```

Comment:
Given this PowerShell property `$host.UI.SupportsVirtualTerminal` and the [MSDN docs](https://msdn.microsoft.com/en-us/library/windows/desktop/mt638032(v=vs.85).aspx), I'm inclined to leave this as-is.  However I did start with ANSI escape sequences.  :-)